Subject: Comment out win only qt5 modules
Description: QT5 modules for windows causing compile errors in linux
Author: Richard Leckinger <richard.leckinger@riseup.net>

--- a/gridcoinresearch.pro
+++ b/gridcoinresearch.pro
@@ -7,9 +7,9 @@ CONFIG += no_include_pwd
 CONFIG += thread
 #QT += sql (Future Use)
 QT += core gui network
-QT += qaxcontainer
+#QT += qaxcontainer
 #QT += axcontainer
-QT += axserver
+#QT += axserver
 QT += widgets
 
 
@@ -18,7 +18,7 @@ greaterThan(QT_MAJOR_VERSION, 4) {
     DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0
 }
 
-CONFIG += qaxcontainer
+#CONFIG += qaxcontainer
 CONFIG += exceptions
 
